core-bundle/src/Util/SymlinkUtil.php

```diff
@@ -51,6 +47,9 @@ public static function symlink(string $target, string $link, string $rootDir): v
      */
     public static function validateSymlink(string $target, string $link, string $rootDir): void
     {
+        $target = Path::normalize($target);
+        $link = Path::normalize($link);
```

Comment:
We could move this after the `if ('' === $target) {` condition so we don't have to evaluate it if the first condition matches.

Reply:
Done in 2fcc0ea72ce47cac179639ed9de6b7bb9d7def89.